commands/auth.go

```diff
@@ -146,7 +146,14 @@ To create new contexts, see the help for `+"`"+`doctl auth init`+"`"+`.`, Writer
 // XDG_CONFIG_HOME is not set, use $HOME/.config. On Windows use %APPDATA%/doctl/config.
 func RunAuthInit(retrieveUserTokenFunc func() (string, error)) func(c *CmdConfig) error {
 	return func(c *CmdConfig) error {
-		token := c.getContextAccessToken()
+		token := viper.GetString(doctl.ArgAccessToken)
```

Comment:
Unfortunately this has some unintended consequences.  As the top level `access-token` item in the config file is bound to the flag value, there is no way for us to determine if this is passed on the command line or sourced from the config file for the default context. 

So this breaks initializing a new non-default context without passing the flag. If a default context is already configured, it's token just gets used:

```
$ rm ~/.config/doctl/config.yaml 
$ ./builds/doctl auth init
Please authenticate doctl for use with your DigitalOcean account. You can generate a token in the control panel at https://cloud.digitalocean.com/account/api/tokens

❯ Enter your access token:  ●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●

Validating token... ✔

$ ./builds/doctl auth init --context test
Using token for context test

Validating token... ✔
```

I'm not sure if there is a simple solution here. This may need to wait until we can do deeper changes to how contexts work.